guardian/tagmanager
# Bump Webpack to version 4

## What does this change?

This PR bumps Webpack 4, in order to reduce the number of vulnerabilities in the application. 

`node-sass` is removed (solving some high risk vulnerabilities), while a number of other webpack-related dependencies are bumped. All our babel dependencies have been modernised.

## Why?
The two critical vulnerabilities reported by Snyk are related to Webpack and`svg-sprite` (which is also bumped in this PR), and `css-loader`, which couldn't be bumped without bumping Webpack.

Some Sass was changed in order to satisfy the newer version of `sass-loader`.
## How to test

Run `tagmanager` locally according to the instructions in the readme:
- Do the four scripts in `package.json` all run as expected? (n.b. the dev server _was_ and _remains_ slow to start)
- Can you deploy to CODE and successfully run the application there? 
- Do the styles seem to be working?
